config/routes.rb

```diff
@@ -175,9 +175,12 @@
         resources :projects, controller: "/admin/custom_fields/custom_field_projects", only: %i[index new create]
         resource :project, controller: "/admin/custom_fields/custom_field_projects", only: :destroy
         resources :items, controller: "/admin/custom_fields/hierarchy/items" do
-          get :deletion_dialog, on: :member
-          get :new_child, on: :member, action: :new
-          post :new_child, on: :member, action: :create
+          member do
+            get :deletion_dialog
+            get :new_child, action: :new
+            post :new_child, action: :create
+            put :move
```

Comment:
🟡 WDYT about a PATCH or POST here? We are technically just updating the resource (hence PATCH), OTOH this triggers a lot of side-effects on the other items so POST would also be accurate.

Reply:
I think you have a good point. I was looking at other examples and we have both routes [with put](https://github.com/opf/openproject/blob/dev/config/routes.rb#L509) and [with post](https://github.com/opf/openproject/blob/dev/config/routes.rb#L348).

I'll adjust this.